src/pywrangler/pyspark/wranglers/interval_identifier.py

```diff
@@ -539,7 +539,7 @@ def _generate_raw_iids_special(self, start_first: bool,
             ff_window)
 
         # shifting marker_col forward
-        shift_col = F.lag(forward_fill_col, default=default, count=1) \
```

Comment:
May we want to ensure backwards compatibility for `pyspark < 3.0.0` which I think is still reasonably. Hence, we will need to provide a compatibility layer around the changing keyword arguments.

Reply:
For line 542 I've removed the problematic keyword entirely, as `count=1` was the default (see e.g. [here](https://spark.apache.org/docs/2.4.8/api/python/pyspark.sql.html?highlight=lag#pyspark.sql.functions.lag)) anyways, and `offset=1` is the current default (see [here](https://spark.apache.org/docs/latest/api/python/reference/pyspark.sql/api/pyspark.sql.functions.lag.html?highlight=lag#pyspark.sql.functions.lag))